doc/rados/operations/pools.rst

```diff
@@ -6,33 +6,33 @@ When you first deploy a cluster without creating a pool, Ceph uses the default
 pools for storing data. A pool provides you with:
```

Comment:
Can you point out the added newline?  I can't pick it out from the whitespace changes (tried twice)